marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/impl/hub-entities.xqy

```diff
@@ -235,6 +237,28 @@ declare %private function hent:fix-options-exp($nodes as node()*, $sortable-prop
       default return $n
 };
 
+declare %private function hent:add-sort-operators-to-search-options($sortable-properties as map:map)
+{
+  let $sort-operators :=
+    for $property-path in map:keys($sortable-properties)
+      let $property-path := fn:replace($property-path, "es:", "*:")
+      let $constraint-name := fn:concat("sort", xdmp:initcap(fn:tokenize($property-path, "/")[last()]))
+      return
+        <search:operator name="{$constraint-name}">
```

Comment:
We only need a single operator with a name of "sort". Then, for every sortable property, we have two states - "(propertyName)Ascending" and "(propertyName)Descending". 

I believe you can get away with "propertyName" and not property path because for 5.3.0, we're only supporting sorting on top-level properties. Thus, we do not need to worry about property paths (we may never have to).